DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -172,6 +172,40 @@ public void Initial_menu_should_have_centre_name()
             initialMenuViewModel.CentreName.Should().Be(centreName);
         }
 
+        [Test]
+        public void Initial_menu_should_have_true_show_time()
+        {
+            // Given
+            const string courseSettings = "{\"lm.st\":true}"; // ShowTime = true
+            var expectedCourseContent = CourseContentHelper.CreateDefaultCourseContent(
+                courseSettings: courseSettings
+
+            );
+
+            // When
+            var initialMenuViewModel = new InitialMenuViewModel(expectedCourseContent);
+
+            // Then
+            initialMenuViewModel.ShowTime.Should().BeTrue();
+        }
+
+        [Test]
+        public void Initial_menu_should_have_false_show_time()

```

Comment:
What about having a test for false show time if the time is null? 🙂

Reply:
Yes we definitely need one. Added in b172872 🙂